static/src/javascripts/projects/common/modules/commercial/creatives/revealer.js

```diff
@@ -24,12 +25,17 @@ define([
                 }
             }).then(function () {
                 return fastdom.read(function () {
-                    return $adSlot[0].getBoundingClientRect();
+                    return detect.getViewport();
+                });
+            }).then(function (viewport) {
+                return fastdom.write(function () {
+                    var background = $adSlot[0].getElementsByClassName('creative__background')[0];
+                    background.style.width = viewport.width + 'px';
+                    // for the height, we need to account for the height of the location bar, which
+                    // may or may not be there. 70px padding is not too much.
+                    background.style.height = (viewport.height + 70) + 'px';
+                    return true;
```

Comment:
I am not sure if I understand how this works. Why do you need to `return true`? Is this method called in here https://github.com/guardian/frontend/blob/2a4f00ba2e7c5b7abd1fff614b03cd7cd4947fea/static/src/javascripts/projects/common/modules/commercial/dfp/private/apply-creative-template.js#L104 ?